rest/api_test.go

```diff
@@ -2157,29 +2157,41 @@ func TestRoleAccessChanges(t *testing.T) {
 
 	// Check user access:
 	alice, _ = a.GetUser("alice")
-	assert.DeepEquals(t, alice.InheritedChannels(), channels.TimedSet{"!": channels.NewVbSimpleSequence(0x1), "alpha": channels.NewVbSimpleSequence(0x1), "gamma": channels.NewVbSimpleSequence(0x1)})
-	assert.DeepEquals(t, alice.RoleNames(), channels.TimedSet{"bogus": channels.NewVbSimpleSequence(0x1), "hipster": channels.NewVbSimpleSequence(0x1)})
+	assert.DeepEquals(t,
+		alice.InheritedChannels(),
+		channels.TimedSet{
+			"!":     channels.NewVbSimpleSequence(0x1),
+			"alpha": channels.NewVbSimpleSequence(0x1),
+			"gamma": channels.NewVbSimpleSequence(0x1),
+		},
+	)
+	assert.DeepEquals(t,
+		alice.RoleNames(),
+		channels.TimedSet{
+			"bogus":   channels.NewVbSimpleSequence(0x1),
+			"hipster": channels.NewVbSimpleSequence(0x1),
+		},
+	)
 	zegpold, _ = a.GetUser("zegpold")
-	assert.DeepEquals(t, zegpold.InheritedChannels(), channels.TimedSet{"!": channels.NewVbSimpleSequence(0x1), "beta": channels.NewVbSimpleSequence(0x1)})
+	assert.DeepEquals(t,
+		zegpold.InheritedChannels(),
+		channels.TimedSet{
+			"!":    channels.NewVbSimpleSequence(0x1),
+			"beta": channels.NewVbSimpleSequence(0x1),
+		},
+	)
 	assert.DeepEquals(t, zegpold.RoleNames(), channels.TimedSet{})
 
-	// Check the _changes feed:
-	var changes struct {
-		Results  []db.ChangeEntry
-		Last_Seq interface{}
-	}
-	response = rt.Send(requestByUser("GET", "/db/_changes", "", "alice"))
-	log.Printf("1st _changes looks like: %s", response.Body.Bytes())
-	json.Unmarshal(response.Body.Bytes(), &changes)
-	assert.Equals(t, len(changes.Results), 2)
-	since := changes.Last_Seq
+	changesResult, err := rt.WaitForChanges(2, "/db/_changes", "alice", false)
```

Comment:
It's probably fine for this test, but in general WaitForSequence feels like a better choice than WaitForChanges, as there's less impact on SG state.  (e.g. the authentication on each changes request could modify the user document,  and so WaitForChanges would hide an issue where the auth is expecteded to work the first time, but instead only works on the second time).